### PR TITLE
[RFC] Add the infosheet

### PR DESCRIPTION
**Pull request by Hjdskes**
~~We need to add the outlook once the recommendations section has been written. Perhaps ask Eelco / Hendrik is the product is going to be used, too.~~

~~We need to ask Hendrik if we can include his email address as well.~~

### CODE REVIEW

File: final/main.tex [hunk 1]
```diff
@@ -122,12 +122,16 @@
...
 
+\input{infosheet}
+
 \input{project-description}
 
 \input{research-report}
 
 \input{full-uml}
 
+\input{research-report}
```
Comment (Balletie) [inline final/main.tex]: ?

Comment (Hjdskes) [inline final/main.tex]: Wrong rebase I guess :P

Comment (Balletie) [inline final/main.tex]: Easy way for giving the page count a little bump.

File: final/infosheet.tex [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+halfway through the project. This new version of DynSem did not expose the
+required functionality for a REPL. Collaborating with its developer, these
+features were implemented over the course of two weeks.
+
+The delivered product consists of a backend, providing a means of parsing,
+analyzing and evaluating expressions using Spoofax's services. Two frontends
+were delivered: one running in the console and the other inside the Eclipse IDE.
+Both the frontends and the backend are tested with unit- and integration tests.
+
+TODO: Outlook: Describe the outlook of the product. Did the team make recommendations to the
```
Comment (Hjdskes) [inline final/infosheet.tex]: We need to have a short meeting with Eelco today and discuss this.

File: final/infosheet.tex [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+\noindent All team members contributed to the reports and the final
+presentation.\\
+
+\noindent\textbf{Client:} Eelco Visser, SLDE group TU Delft\\
+\textbf{Coach:} Hendrik van Antwerpen, SLDE group TU Delft\\
+\textbf{Contact:}
+\href{mailto:gerlof.fokkema@gmail.com}{gerlof.fokkema@gmail.com},
+\href{mailto:hjdskes@gmail.com}{hjdskes@gmail.com},
+\href{mailto:skippetie@gmail.com}{skippetie@gmail.com}\\
```
Comment (Hjdskes) [inline final/infosheet.tex]: We need to ask Hendrik if we can add his email address here as well.

**Comment by Hjdskes** [comment]
~~To add: the project is difficult because no assumptions can be made regarding host languages.~~ Fixed in b257e6a015b02870eccf0b6bc7a4aa37f0f9685c.

File: final/infosheet.tex [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+\subsection*{Client:}
+Eelco Visser, SLDE group TU Delft
+
+\subsection*{Coach:}
+Hendrik van Antwerpen, SLDE group TU Delft
+
+\subsection*{Contact:}
+Gerlof Fokkema, \href{mailto:gerlof.fokkema@gmail.com}{gerlof.fokkema@gmail.com}\\
+Jente Hidskes, \href{mailto:hjdskes@gmail.com}{hjdskes@gmail.com}\\
+Skip Lentz, \href{mailto:skippetie@gmail.com}{skippetie@gmail.com}\\
```
Comment (Unia) [inline final/infosheet.tex]: We need to ask Hendrik if we can add his contact details as well.

**Comment by Hjdskes** [comment]
Please merge this after #84.

**Pull request merged**